DQMOffline/Trigger/python/BPHMonitor_cff.py

```diff
@@ -3,34 +3,328 @@
+Dimuon0_dRcut_low.numGenericTriggerEventPSet.l1Algorithms = cms.vstring("DoubleMu4_SQ_OS_dR_Max1p2")
+Dimuon0_dRcut_low.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon0_LowMass_L1_4_v1")
+Dimuon0_dRcut_low.denGenericTriggerEventPSet.l1Algorithms = cms.vstring("DoubleMu4_SQ_OS")
+Dimuon0_dRcut_low.muoSelection_ref = cms.string("abs(eta)<2.4 &  isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+Dimuon0_dRcut_low.muoSelection = cms.string("abs(eta)<2.4 & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+###
+###mass cut
+Dimuon0_masscut1 = hltBPHmonitoring.clone()
+Dimuon0_masscut1.FolderName = cms.string('HLT/BPH/DiMu0_Lowmass_L1_masscut1/')
+Dimuon0_masscut1.tnp = cms.int32(0)
+Dimuon0_masscut1.nofset = cms.int32(5)
+Dimuon0_masscut1.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon0_Upsilon_L1_4p5er2p0M_v1")
+Dimuon0_masscut1.numGenericTriggerEventPSet.l1Algorithms = cms.vstring("DoubleMu4p5er2p0_SQ_OS_Mass_7to18")
+Dimuon0_masscut1.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon0_Upsilon_L1_4p5er2p0_v1")
+Dimuon0_masscut1.denGenericTriggerEventPSet.l1Algorithms = cms.vstring("DoubleMu4p5er2p0_SQ_OS")
+Dimuon0_masscut1.muoSelection_ref = cms.string("abs(eta)<2. &  isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+Dimuon0_masscut1.muoSelection = cms.string("abs(eta)<2. & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+
+Dimuon0_masscut2 = hltBPHmonitoring.clone()
+Dimuon0_masscut2.FolderName = cms.string('HLT/BPH/DiMu0_Lowmass_L1_masscut2/')
+Dimuon0_masscut2.tnp = cms.int32(0)
+Dimuon0_masscut2.nofset = cms.int32(5)
+Dimuon0_masscut2.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon0_Upsilon_L1_5M_v1")
+Dimuon0_masscut2.numGenericTriggerEventPSet.l1Algorithms = cms.vstring("DoubleMu5_SQ_OS_Mass_7to18")
+Dimuon0_masscut2.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon0_Upsilon_L1_5_v1")
+Dimuon0_masscut2.denGenericTriggerEventPSet.l1Algorithms = cms.vstring("DoubleMu5_SQ_OS")
+Dimuon0_masscut2.muoSelection_ref = cms.string("abs(eta)<2.4 &  isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+Dimuon0_masscut2.muoSelection = cms.string("abs(eta)<2.4 & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+###triple muon
+
+Dimuon0_tripleMu1 = hltBPHmonitoring.clone()
+Dimuon0_tripleMu1.FolderName = cms.string('HLT/BPH/DiMu0_Lowmass_L1_tripleMu1/')
+Dimuon0_tripleMu1.tnp = cms.int32(0)
+Dimuon0_tripleMu1.nofset = cms.int32(6)
+Dimuon0_tripleMu1.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Trimuon2_Upsilon5_Muon_NoL1Mass_v1")
```

Comment:
I guess you are referring to the "_NoL1Mass" part.
This path is identical to the physics one (HLT_Trimuon2_Upsilon5_Muon_v*) and seeded by the same L1 seed, except the removed L1 mass cut.
Adding "_NoL1Mass" seemed to me the most clear and short way to reflect this.
A better naming convention is welcomed. In case please comment the JIRA ticket:
https://its.cern.ch/jira/browse/CMSHLT-1299